ux.symfony.com/cookbook/component_architecture.md

```diff
@@ -174,5 +174,5 @@ class Button
 ## Conclusion
 
 Even in Symfony, you can use the component architecture.
-Follow those rules help your front developpers working on codebase
-their are familiar with since those rules are already used in the js world.
+Follow those rules help your front developers working on codebase
+they are familiar with since those rules are already used in the JS world.
```

Comment:
`JS` or `JavaScript`?

Reply:
Quick fixes as i saw them.

 Could you open a PR/issue if you review the content (grammar / typos / style) ?